### ISSUE TITLE
/server uses a vulnerable version of node-fetch and should be patched >= 2.6.7

### ISSUE THREAD

**Issue by eddie-matsec** [MasterKale/SimpleWebAuthn#186]
I see [recent changes](https://github.com/MasterKale/SimpleWebAuthn/pull/175) in /example were made to resolve node-fetch to >= 2.6.7 ([CVE-2022-0235](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-0235))

Can we apply the same for /packages/server so that it includes `node-fetch@^2.6.7`?

**simplewebauthn/packages/server/package.json**
```
"dependencies": {
    .
    .
    .
    "node-fetch": "^2.6.0",
  },
```

**Comment by MasterKale** [comment]
Since PR #68 these libraries specifically use carets for versioning so that projects using the library remain free to incorporate security fixes for things like `node-fetch` without my needing to cut a new release. It's also why I don't include **package-lock.json** files in server or browser, unlike /example which I'm free to update via `npm audit fix` just like any other application that consumes this library.

Are you unable to run `npm audit fix` and update your project to node-fetch@2.6.7+ because of **simplewebauthn/packages/server/package.json**?

**Comment by MasterKale** [comment]
I'm going to close this out for lack of feedback. Please feel free to reopen if you believe it's still an issue.
